README.md

```diff
@@ -28,6 +28,12 @@
 
 [![License](https://img.shields.io/badge/License-BSD3-lightgrey.svg)](https://opensource.org/licenses/BSD-3-Clause)
 
+**LATEST RELEASE: You are currently on the main branch which tracks
+under-development progress towards the next release. The current release is
+version [2.39.0](https://github.com/triton-inference-server/vllm_backend/tree/r23.10)
```

Comment:
Is it possible to refer to Triton's latest release page? Otherwise, we would have to update this every month, which can be easily forgotten

Reply:
I would update the release instructions to update this message here. We have the instructions for the server repo already. Including the specific release branch would be useful for the audience to access the instructions in a reliable fashion.